ansible/roles/k3s/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+# Warning: changes to these variables won't be reflected in the cluster/image if k3s is already installed
+k3s_version: "v1.31.0+k3s1"
+k3s_selinux_release: v1.6.latest.1
+k3s_selinux_rpm_version: 1.6-1
+rocky_version: "{{ ansible_distribution_major_version }}"
```

Comment:
Remove this and just use ansible_distribution_major_version in the correct place - this is odd, as it is not a k3s_ prefixed var and I can't see why you would want to override it in its use in ansible/roles/k3s/tasks/main.yml?